guides/integrate-nextauth.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Integrate Beyond Identity Passwordless Authentication into NextAuth
+sidebar_position: 1
+---
+
+# Integrate with NextAuth
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a NextAuth site.
+
+This guide will cover:
+  * How to configure Beyond Identity as an Identity Provider
+  * How to install the required NextAuth plugins to support an OpenID Connect (OIDC) integration
+  * How to configure the OIDC NextAuth plugin
+
+## Prerequisites
+
+Before continuing, make sure that the following prerequisites have been met:
+  * Ensure that you have a live NextAuth site running and administrator privileges to install and configure a plugin
+  * Access to a Beyond Identity tenant and its admin console
+
+## Set up Beyond Identity as an Identity Provider
+
+### Create a Realm and Application in Beyond Identity
+
+### Create a Realm
+
+We need to create a new [Realm](https://developer.beyondidentity.com/docs/v1/platform-overview/architecture#realms) to hold identities and configuration:
+
+<div style={{position: 'relative', paddingBottom: 'calc(73% + 20px)', height: '0'}}>
+	<iframe src='https://demo.arcade.software/eyWvI91g13J7qj5vmCfD?embed&forceNoOpeningAnimation=true' frameBorder="0" style={{position: 'absolute', top: '0', left: '0', width: '100%', height: '100%'}}>
+	</iframe>
+</div>
+
+From the Admin Console:
+  1. Click the Realm selector on the top-left.
+  2. Click **Create New Realm**.
+  3. Type the name of your new realm. Click **Create Realm**.
+  4. Click **Switch to Realm**.
+  5. From the Realm's Home page, click Edit.
+
+### Create an Application
+
+Next, we we'll create a new [Application](https://developer.beyondidentity.com/docs/v1/platform-overview/architecture#applications-and-authenticator-configs) that contains the configuration for your end users:
+
+<div style={{position: 'relative', paddingBottom: 'calc(73% + 20px)', height: '0'}}>
+	<iframe src='https://demo.arcade.software/KmtiNsx4Z31MkogQdwST?embed&forceNoOpeningAnimation=true' frameBorder="0" style={{position: 'absolute', top: '0', left: '0', width: '100%', height: '100%'}}>
+	</iframe>
+</div>
+
+From the Admin Console: 
+  1. From the navigation bar, click **Applications**, then click **Add app**.
+  1. Type a name for your new Application.
+  1. Scroll to **Redirect URIs**, type `http://localhost:3000`.
+  1. Ensure **"Token Endpoint Auth Method"** is set to "Client Secret Post".
+  1. Ensure **"Subject"** is set to "id".
+  1. Verify that **"PKCE"** is disabled.
+  1. Click on the Authenticator Config tab, change the Configuration Type to `Hosted Web`.
```

Comment:
And I think we want Embedded here?

Reply:
I think it might be better to explain the difference, or link to https://developer.beyondidentity.com/docs/v1/platform-overview/authenticator-config, since it depends on what the developer is doing.